src/Javis.jl

```diff
@@ -415,13 +413,14 @@ for func in names(Luxor; imported = true)
 end
 
 export javis, latex
-export Video, Action, BackgroundAction, SubAction, Rel
+export Video, Object, BackgroundObject, Action, Rel
 export Line, Translation, Rotation, Transformation, Scaling
 export val, pos, ang, get_value, get_position, get_angle
-export projection, morph
+export projection, morph_to
```

Comment:
Why the name change for morph?

Reply:
The morph functionality changed into `Action` so it only has one argument now. Therefore it's not `morph(star, circle)` but `morph_to(circle)` if the parent object is a star. Hope that makes sense.

Reply:
:+1: